src/main/java/org/kuali/kfs/module/ar/document/web/struts/PaymentApplicationForm.java

```diff
@@ -145,7 +146,14 @@ && otherAppsAssociatedWithTheCashControlAssociatedWithTheAppAreApprovedOrCancele
     }
 
     protected boolean appHasNoPreviousOrPendingAdjustments() {
```

Comment:
For consistency with what you did on the other class, I would suggest adding a comment to this method indicating that it was customized to include a backport.